assets/colors/colors.ts

```diff
@@ -1,2 +1,8 @@
-export const white = 'white'
-export const peach = '#FD9890'
\ No newline at end of file
+export const white = 'white';
+export const peach = '#FD9890';
+export const red = 'red';
+export const blue = 'blue';
+export const black = 'black';
+export const lightgray = '#E2DFDE';
```

Comment:
Please use camell-case for variable namings.
```suggestion
export const lightGray = '#E2DFDE';
```